docs/development/releasing.md

```diff
@@ -206,7 +206,7 @@ release to npm.
 
 [the releases page]: https://github.com/electron/electron/releases
 [this bump commit]: https://github.com/electron/electron/commit/78ec1b8f89b3886b856377a1756a51617bc33f5a
-[electron-versioning]: /docs/tutorial/electron-versioning.md
+[versioning]: /docs/versioning.md
```

Comment:
I'm happy with dropping the `electron-` part of the filename, but I still think `docs/tutorial` is the right(est) location for this file. I would prefer to reduce the number of files in the top-level of `/docs` rather than grow it.